docs/content/1.getting-started/1.quick-start.md

```diff
@@ -52,6 +52,27 @@ And that's it! The Nuxt Security module will now register routeRoules and middle
    })
    ```
 
+   Or, you can pass options directly
+
+   ```js{}[nuxt.config.js]
+   export default defineNuxtConfig({
+     modules: [
+      [
+        'nuxt-security',
+        {
+          requestSizeLimiter: {
+            value: {
+              maxRequestSizeInBytes: 3000000,
+              maxUploadFileRequestInBytes: 9000000,
+            },
+            route: '/upload-file'
+          }
+          // Other options
+        }
+      ]
+     ]
+   })
+   ```
```

Comment:
suggestion: could you also add it to the README.md? It would be beneficial to have the same docs here and there :)

Reply:
Sure, I have added to README.md.
Please, review my PR again.

Reply:
Thanks, I approve and merge!

Reply:
I will release it as a part of the next release (0.11.0) in the upcoming days :)